spec/deprecation-cop-status-bar-view-spec.coffee

```diff
@@ -65,6 +65,9 @@ describe "DeprecationCopStatusBarView", ->
     runs ->
       expect(statusBarView).toBeVisible()
       expect(statusBarView.textContent).toBe '1'
+    
+      atom.packages.deactivatePackage("theme-with-deprecated-selectors")
+      atom.packages.unloadPackage("theme-with-deprecated-selectors")
```

Comment:
I needed to add this to make this check work correctly: https://github.com/atom/deprecation-cop/pull/44/files#diff-f576378ee9f22cd2607944fad4c7c507R61. It seems that because this spec was executed first, the `theme-with-deprecated-selectors` theme was still loaded when the other spec was executed. Not 100% sure if I got that right or if there's a better way to handle this.